statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -884,7 +887,11 @@ def _get_robustcov_results(self, cov_type='opg', **kwargs):
         res.cov_kwds = {}
 
         # Calculate the new covariance matrix
-        if self.cov_type == 'cs':
+        if len(self.params) == 0:
```

Comment:
There is no particular case when params is empty. In some sense it's still a well-defined problem (i.e. there is a likelihood), and it was pretty trivial to allow empty params (just checks in two places), but mostly it's just hard to test MLEModel by itself (rather than through a subclass) if empty parameters throw errors.